config/default.yml

```diff
@@ -711,6 +711,17 @@ PreCommit:
     install_command: 'npm install -g standard'
     include: '**/*.js'
 
+  StyleLint:
+    enabled: true
```

Comment:
We do not enable hooks by default unless they are truly applicable to almost any project. Set this to `false`.